pandas/io/formats/html.py

```diff
@@ -161,15 +161,7 @@ def write_result(self, buf):
             _classes.extend(self.classes)
 
         if self.notebook:
-            div_style = ''
-            try:
-                import IPython
```

Comment:
The `import Python` was causing a failure. I figure we're OK to drop Python 2.x compat code :)